### PR TITLE
Evaluator run test repeatedly

### LINKED ISSUE #214: Make evaluator tests pass in SDK
Remove library dependencies like whitenoise-core that not allowing build to work. Remove any tests that are probabilistic and don't pass every time like the bias test. Make verification files internal. This'll help get a build setup.

### PR DESCRIPTION
**Pull request by ankit-oss**
Fixes #214 

Result of running same test repeatedly fails bias test for SUM and other operators. So disabling it as it's a known issue. Also, the mechanism tests are not needed to run as a BVT. So we can skip those. 

============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 125.04s (0:02:05) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 101.57s (0:01:41) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======= 3 passed, 5 skipped, 3 deselected, 1 warning in 94.00s (0:01:33) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 116.91s (0:01:56) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 100.42s (0:01:40) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 108.92s (0:01:48) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 105.63s (0:01:45) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 100.14s (0:01:40) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======= 3 passed, 5 skipped, 3 deselected, 1 warning in 95.34s (0:01:35) =======
============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.3.1, py-1.8.0, pluggy-0.13.1
rootdir: /mnt/c/Users/ankitsri/source/code/privacytoolsproject/whitenoise-system/tests, inifile: pytest.ini
collected 11 items / 3 deselected / 8 selected

test_stochastic.py ...sssss                                              [100%]

=============================== warnings summary ===============================
/mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323
  /mnt/c/Users/ankitsri/source/code/miniconda3/envs/oss_dp/lib/python3.8/site-packages/_pytest/mark/structures.py:323: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 3 passed, 5 skipped, 3 deselected, 1 warning in 100.77s (0:01:40) =======

### CODE REVIEW

File: tests/evaluation/test_stochastic.py [hunk 1]
```diff
@@ -92,7 +94,7 @@ def test_powerset_sum(self):
         assert(dp_res)
         #assert(acc_res)
         #assert(utility_res)
-        assert(bias_res)
+        #assert(bias_res)
```
Comment (eedeleon) [inline tests/evaluation/test_stochastic.py:97]: no need to comment out, skip will do the trick. That way when we remove skip we get the expected behavior

File: tests/evaluation/test_stochastic.py [hunk 2]
```diff
@@ -106,7 +108,7 @@ def test_groupby(self):
...
```
Comment (eedeleon) [inline tests/evaluation/test_stochastic.py:111]: same idea for these

File: tests/evaluation/test_stochastic.py (hunk 1 above)
Comment (ankit-oss) [reply tests/evaluation/test_stochastic.py]: Good point. Made slow tests also skip. Also removed any commented asserts. Bias test is not expected to pass anytime soon. So removing it is fine.

File: tests/evaluation/test_stochastic.py (hunk 2 above)
Comment (ankit-oss) [reply tests/evaluation/test_stochastic.py]: Removed the commented asserts as we know they fail and we can bring them back when Unbiased Privacy is available.

**Comment by ankit-oss** [comment]
Thank you!

**Pull request merged**